### ISSUE TITLE
New Voucher price field required?

### ISSUE THREAD

**Issue by smockensturm** [verbb/gift-voucher#80]
could this be a little more graceful please?

<img width="1107" alt="Screen Shot 2021-12-07 at 2 51 11 PM" src="https://user-images.githubusercontent.com/6586518/145097013-6e4af138-47b6-4c1c-aee0-c72b8f9cdbd5.png">

**Comment by engram-design** [comment]
Yeah, not sure why that's started throwing that hard-error. Fixed for the next release. To get the fix early, change your `verbb/gift-voucher` requirement in `composer.json` to:

```
"require": {
  "verbb/gift-voucher": "dev-craft-3 as 2.6.3",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [2.7.0](https://github.com/verbb/gift-voucher/releases/tag/2.7.0)
